chia/cmds/wallet_funcs.py

```diff
@@ -94,9 +94,13 @@ async def send(args: dict, wallet_client: WalletRpcClient, fingerprint: int) ->
 
 async def get_address(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
     wallet_id = args["id"]
-    res = await wallet_client.get_next_address(wallet_id, False)
-    print(res)
-
+    new_address = args["new_address"]
+    if new_address == True:
+        res = await wallet_client.get_next_address(wallet_id, True)
+        print(res)
+    else:
+        res = await wallet_client.get_next_address(wallet_id, False)
+        print(res)
```

Comment:
These 6 lines would be the same as
```python
res = await wallet_client.get_next_address(wallet_id, new_address)
print(res)
```
For these flags I'd also use a verb, otherwise the variable name `new_address` sounds like its value is a new address.

Reply:
Thank you for the review, i changed the files according to your suggestion.